tests/core/tools/manual/escapecss.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.16.2, bug, 681
+
+1. Select the table cell in IE and open developer console.
```

Comment:
Please, separate this step on two separate one:
- opening dev console
- selecting cells

If we want to restrict this test only to IE, all other browsers should be ignored in `html` file. For example, it is done like:
https://github.com/ckeditor/ckeditor4/blob/9c09a211f287e135223d4d2056959106d389c84b/tests/core/ckeditor/manual/hcdetection.html#L24-L26

Please adjust `html` file of this test to ignore non-IE browsers. Also, you can skip IE part in the scenario step description.